Makefile

```diff
@@ -205,26 +154,20 @@ else
 		--non-interactive \
 		--token ${FIREBASE_TOKEN} \
 		--project ${FIREBASE_ALIAS} \
-		--debug \
-		--json
```

Comment:
Why was this specifying `--json` before?

Reply:
I had added it long ago for CI purposes, but the regular output is better I feel.

Reply:
understood!